charts/nodejs/templates/_helpers.tpl

```diff
@@ -7,7 +7,11 @@ Expand the name of the chart.
 {{- end -}}
 
 {{- define "appname" -}}
+{{- if (eq .Values.kubernetes.env "production") -}}
 {{- printf "%s" .Release.Name | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- printf "%s-%s" .Release.Name .Values.biomageCi.sandboxId | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
```

Comment:
Right now, the app name is simply the name of the release (`ui`). If in staging, this is now going to be (`ui-asd`) where `asd` is the ID of the sandbox environment.